### PR TITLE
minor change for example

### PR DESCRIPTION
**Pull request by wlw1227**
duplicated thing for example

### CODE REVIEW

File: app/models/article.rb [hunk 1]
```diff
@@ -22,4 +22,5 @@ def owned_by?(owner)
     return false unless owner.is_a?(User)
     user == owner
   end
+
```
Comment (amyhenning) [inline app/models/article.rb]: nit: can remove this new line

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <%= link_to "Captain's Blog", root_path, class: "px-4 py-2 mr-4 text-lg font-medium" %>
     <%= link_to 'My Articles', articles_path, class: "px-2 py-2 mr-4 text-lg font-medium  rounded-md  hover:bg-gray-200" %>
     <%= link_to 'New article', new_article_path, class: "px-2 py-2 mr-4 text-lg font-medium  rounded-md  hover:bg-gray-200" %>
+    <%= link_to 'New article', new_article_path, class: "px-2 py-2 mr-4 text-lg font-medium  rounded-md  hover:bg-gray-200" %>
```
Comment (amyhenning) [inline app/views/layouts/_header.html.erb]: This looks like a duplicate. Why is it being added?

**Review by amyhenning — COMMENTED** [review]
Left a couple questions.

File: app/models/article.rb (hunk 1 above)
Comment (wlw1227) [reply app/models/article.rb]: removed duplicate line

Comment (amyhenning) [reply app/models/article.rb]: I still see it in here, have you committed the change to remove it?

Comment (wlw1227) [reply app/models/article.rb]: removed new line

**Review by amyhenning — APPROVED** [review]
Looks good to me.

**Pull request merged**